multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/services/ProcessLogger.java

```diff
@@ -91,4 +84,16 @@ public synchronized void deleteLogFile() {
         FileUtils.deleteQuietly(log);
     }
 
+    public String getLoggerName() {
+        return this.logger.getName();
+    }
+
+    public LoggerContext getLoggerContext() {
+        return this.loggerContext;
+    }
+
+    public void closeLoggerContext(){
+        loggerContext.close();
```

Comment:
Is it possible to throw exceptions during close method call?

Reply:
Yes. It is possible and it will be surrounded by try-catch.